src/Microsoft.DotNet.Wpf/src/UIAutomation/UIAutomationTypes/Resources/xlf/Strings.zh-Hans.xlf

```diff
@@ -74,12 +74,12 @@
       </trans-unit>
       <trans-unit id="LocalizedControlTypeHyperlink">
         <source>hyperlink</source>
-        <target state="translated">超链接</target>
+        <target state="translated">hyperlink</target>
```

Comment:
The three that I see are hyperlink, image, and table.  These should remain localized.  Thanks.